README.md

```diff
@@ -66,6 +66,38 @@ $ npm run dev
 
 And boom! Navigate to `http://localhost:3000/` to see the app.
 
+## Development using Dev Container (Optional) 🐳
+
+We have a development container to help you quickly set up your dev environment. This is completely optional but can make getting started easier.
+
+### Requirements
+
+- [Docker](https://www.docker.com/products/docker-desktop)
+- [VS Code](https://code.visualstudio.com/)
+- [Remote - Containers extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+
+### Steps
+
+1. **Clone the repository**
+    ```sh
+    git clone [your-repository]
```

Comment:
This should be updated with the `git` repository instead of `[your-repository]`

Reply:
Fixed